packages/familie-tidslinje/package.json

```diff
@@ -29,9 +29,12 @@
         "nanoid": "^5.0.4"
     },
     "peerDependencies": {
-        "@types/react": "17.x || 18.x",
-        "react": "17.x || 18.x",
-        "styled-components": "5.x | 6.x"
+        "@types/react": "18.x",
+        "react": "18.x",
+        "styled-components": "5.x || 6.x",
+        "@navikt/ds-css": "5.x",
+        "@navikt/ds-react": "5.x",
+        "dayjs": "1.x"
```

Comment:
Denne komponenten bruker dayjs og bør derfor ha denne som peerDependency

Reply:
Denne dependencien må inn i under devDependencies

Reply:
Bra catch - fixed ✅

Reply:
Vi har skrevet om fra dayjs til datefns i ks og ba (fordi det er det dato-komponentene fra aksel bruker), hadde vært digg å skrive om her også egentlig 🤩 men det kan vi ta senere

Reply:
Hadde det vært en ide å legge den til i dependencies-listen i stedet for?

Reply:
Skjønner.
`familie-tidslinje` bruker dayJs ca. 15 steder internt i komponentene sine, så det må isåfall byttes i en annen PR, ja. 

[Eksempel](https://github.com/navikt/familie-felles-frontend/blob/1c15c066a426d04ec7155ef734d6db274c382b9d/packages/familie-tidslinje/src/components/types.internal.ts#L4)